<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateEditWidget</class>
 <widget class="QWidget" name="CoordinateEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtX"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtY"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtZ"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
